pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -105,4 +105,6 @@ stdenv.mkDerivation rec {
     license = with licenses; [ gpl2Only lgpl3Plus gpl3Plus ];
     maintainers = with maintainers; [ eelco r-burns ];
   };
-}
+} // lib.optionalAttrs (stdenv.targetPlatform.useLLVM or false) {
+  NIX_CFLAGS_COMPILE = "-Wno-unused-private-field";
+})
```

Comment:
I think it'd be better to put these in `env.NIX_CFLAGS_COMPILE`; putting it visibly "inside' the drv rather than down at the bottom.

Reply:
Wouldn't that affect the rebuild count?

Reply:
can add it to configure flags. seems like `stdenv.cc.isClang` might be a better fit for a test as this effects things compiled with clang, not just stdenvs with the full llvm toolchain.
`configureFlags = [...] ++ lib.optional stdenv.cc.isClang "CFLAGS=-Wno-unused-private-field";`

if configure is not cooperating then can do `env = lib.optionaAttrs stdenv.cc.isClang { NIX_CFLAGS_COMPILE = "the flag";};`

Reply:
Done